packages/clients/src/helpers/marshalling.ts

```diff
@@ -41,9 +41,11 @@ export const resolveOneOf = <T>(
   }[],
   isRequired = false,
 ): Record<string, T> => {
-  const elt = list.find(obj => obj.value) || list.find(obj => obj.default)
-  const value = elt?.value || elt?.default
-  if (value) return { [elt.param]: value }
+  const elt =
+    list.find(obj => obj.value !== undefined) ||
```

Comment:
```suggestion
    list.find(obj => obj.value !== undefined) ??
```

If the value found is `false`, we want to keep it

Reply:
Je me demande comment le test actuel passe en ayant oublié ça mmm